site/docs/android-ndk.md

```diff
@@ -255,6 +255,77 @@ cc_library(
 )
 ```
 
+## Integration with platforms and toolchains
+
+Bazel's configuration model is moving towards 
+[platforms](https://docs.bazel.build/versions/master/platforms.html) and 
+[toolchains](https://docs.bazel.build/versions/master/toolchains.html). If your
+build uses the `--platforms` flag to select for the architecture or operating system
+to build for, you will need to pass the `--extra_toolchains` flag to Bazel in
+order to use the NDK.
+
+For example, to integrate with the `android_arm64_cgo` toolchain provided by
+the Go rules, pass `--extra_toolchains=@androidndk//:all` in addition to the
+`--platforms` flag.
+
+```
+bazel build //my/cc:lib \
+  --platforms=@io_bazel_rules_go//go/toolchain:android_arm64_cgo \
+  --extra_toolchains=@androidndk//:all
+```
+
+You can also register them directly in the `WORKSPACE` file:
```

Comment:
Why can't the existing android_ndk_repository rule handle this directly?

Reply:
How do you register toolchains from a native RepositoryFunction?

Reply:
There's no direct native equivalent to `register_toolchains`, but it wouldn't be hard to add. I feel like the improved usability would be helpful (but this could be in a separate change).